### ISSUE TITLE
How to train image alignment model?

### ISSUE THREAD

**Issue by Zxkai** [hmf21/AerialVL#2]
Thank you for providing this dataset!!. 
However, I have a question about its suitability for training of image alignment model, which requires image pairs that can be perfectly aligned. From what I’ve observed, it seems that the dataset might not include such image pairs, or at least they are not explicitly labeled for this purpose.
Can you confirm whether the dataset includes data suitable for image alignment model, or if not, whether you have any recommendations for how can I adapt it for this?

**Comment by hmf21** [comment]
Hi @Zxkai ,
Thanks for your concerns, and your suggestion is indeed helpful.
Fot the dataset we uploaded, it contains the dataset for the VPR model training, which is also presented in our manuscript.
And we think this dataset can be also used to train the alignment model as you have suggested before.
For example, we can choose the tiles from the same places at different years to make the image pairs perfectly aligned.
Motivated by your suggestions, we have also uploaded the raw satellite imageries collect from the [USGS](https://earthexplorer.usgs.gov/), which can be esaily re-organized as a dataset for the alignment model training.
